src/proxy.cpp

```diff
@@ -66,6 +66,8 @@
     delete poller_receive_blocked;\
     delete poller_send_blocked;\
     delete poller_both_blocked;\
+    delete poller_frontend_only;\
+    delete poller_backend_only\
```

Comment:
@bluca Hm, yes, I deliberately removed it because macro is called with a ";" at the end, so it doesn't look odd in the code.
What is the rule for this, I can return the ";"?